projects/lib/src/chessgame.cpp

```diff
@@ -647,6 +660,16 @@ void ChessGame::pause()
 	m_paused = true;
 }
 
+void ChessGame::halt()
```

Comment:
Looks like this method is not used anywhere. Is it needed in the future?

Reply:
Okay. I'd rather avoid committing unused code. But you can always add a TODO comment about planned improvements to the code.

Reply:
Unused code removed